dbtmetabase/__init__.py

```diff
@@ -78,58 +139,103 @@ def main(args: list = None):
     )
     parser.add_argument(
         "--mb_https",
-        metavar="HTTPS",
-        type=bool,
-        default=True,
-        help="use HTTPS to connect to Metabase instead of HTTP",
+        action="store_true",
+        default=False,
```

Comment:
Is the change from `True` to `False` intentional?

Reply:
No, I do generally think store_true & store_false is a little more readable / understandable compared to a flag that needs a string arg which evals to a bool but for the sake of consistency in expected function, I regressed this change.